usaspending_api/accounts/models/historical_appropriation_account_balances.py

```diff
@@ -55,7 +55,7 @@ class HistoricalAppropriationAccountBalances(models.Model):
     reporting_fiscal_quarter = models.IntegerField(blank=True, null=True)
     reporting_fiscal_period = models.IntegerField(blank=True, null=True)
 
-    owning_toptier_agency = models.ForeignKey("references.ToptierAgency", models.DO_NOTHING, null=True)
+    owning_toptier_agency = models.ForeignKey("references.ToptierAgency", models.CASCADE, null=True)
```

Comment:
Concerned that making this change will result in data being silently deleted from this table in the future, should we decide to make use of it. My understanding is that this case does not occur very often and could be handled manually each time or via something such as `models.SET` (https://docs.djangoproject.com/en/2.2/ref/models/fields/#django.db.models.SET).

Reply:
Additionally, this table contains only data from FY14 through FY17. This means it is even more likely to be silently deleted since we would not have a step in our nightly pipeline to refresh it each night.